BranchMetrics/ios-branch-deep-linking-attribution
# [SDK-2068] Added ScCid support and tests

## Reference
SDK-2068 -- Collect, Persist, and Forward 'ScCid' value

## Summary
Added support for collecting the Snapchat Click Identifier (ScCid) as a URL Query parameter. The parameter is collected and attached to each install, open, and v2/event request. Since ScCid is is session based, it is cleared whenever the app is backgrounded or terminated. 

## Motivation
To collect and forward the new ScCid URL query parameter.

## Type Of Change


## Testing Instructions

Open a link like `https://branchtestbed.app.link/testlink&ScCid=c22f092d-0961-4dff-9110-009df25e1328` and observe that the ScCid is persistent and attached to the proper requests, then cleared appropriately. 









cc @BranchMetrics/saas-sdk-devs for visibility.
